ciscoise.json

```diff
@@ -801,13 +925,866 @@
+                },
+                {
+                    "data_path": "action_result.data.*.id",
+                    "data_type": "string",
+                    "example_values": [
+                        "40963c00-2e02-11e8-ba71-005056872c7f"
+                    ],
+                    "column_name": "Resource ID",
+                    "column_order": 0,
+                    "contains": [
+                        "ise resource id"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.link.href",
+                    "data_type": "string",
+                    "example_values": [
+                        "https://10.1.16.169:9060/ers/config/portal/40963c00-2e02-11e8-ba71-005056872c7f"
+                    ],
+                    "contains": [
+                        "url"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.link.rel",
+                    "data_type": "string",
+                    "example_values": [
+                        "self"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.link.type",
+                    "data_type": "string",
+                    "example_values": [
+                        "application/xml"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "Sponsor Portal (default)"
+                    ],
+                    "column_name": "Resource Name",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.summary.resources_returned",
+                    "data_type": "numeric",
+                    "example_values": [
+                        5
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "Resources returned: 5"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get resources",
+            "description": "Search for resources. Returns resource with `resource id`, else finds resources with key-value",
```

Comment:
Can we change it with "Get the information about resource if resource_id is provided. Fetch the list of resources match with the key-value filter"

Reply:
Updated